src/TritonRoute/src/db/obj/frShape.h

```diff
@@ -566,7 +566,11 @@ class frPathSeg : public frShape
   }
   void setTapered(bool t) { tapered_ = t; }
   bool isTapered() const { return tapered_; }
-
+  
+  bool intersectsCenterLine(const frPoint& pt) {
```

Comment:
This does not consider the end extension style.  Is that intentional?

Reply:
Yes.